etna/records/models.py

```diff
@@ -83,6 +83,41 @@ def get_gtm_content_group(self) -> str:
         else:
             return self.__class__.__name__
 
+    @property
+    def _customdimension11(self) -> str:
```

Comment:
Do you not like the format `custom_dimension_11`?

Reply:
Done :)